src/scrapers/visa-cal.ts

```diff
@@ -133,6 +135,8 @@ function convertTransactionType(txnType: string) {
     case WITHDRAWAL_TYPE_CODE_2:
     case REFUND_TYPE_CODE_2:
     case CANCEL_PAYMENT_CODE:
+    case CANCELLED_TRANSACTION:
+    case TYPE_CODE_1:
```

Comment:
@esakal I don't like the fact we solve this for a type we don't know
it will be just like whitelisting all types

Reply:
I agree, but I was trying to help #562 which has this issue but since they opened the ticket they didn't revisit to answer my question.

If you want feel free to remove it so we will be able to merge the support for 68. I would do that but since you are planning on releasing later today we can handle that one at the same time